backend/server/sponsor.go

```diff
@@ -19,7 +22,7 @@ type Sponsor struct {
 	Name   string `json:"name" validate:"required"`
 	Logo   string `json:"logo" validate:"required,url"`
 	Tier   int    `json:"tier" validate:"required,numeric,eq=10|eq=100|eq=1000"`
```

Comment:
Hmm maybe we can use strings to represent principal sponsor, etc instead of type int to make it more clear. What do you think? @glebme

Reply:
I prefer int since there won't be spelling mistakes and is already how it is being done from the front end. The point on just who's principal etc and how it should be displayed is a front end matter that they can deal with.

Reply:
@boboeee Did you post the question here?